pootle/apps/pootle_store/diff.py

```diff
@@ -32,6 +32,27 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def hasplural(self):
+        return (
+            self.source is not None
+            and (len(self.source.strings) > 1
+                 or getattr(self.source, "plural", None)))
+
+    def getnotes(self, origin=None):
+        return self.unit["%s_comment" % origin]
```

Comment:
How does this render in case no `origin` is specified? Perhaps it is better not to default to `None`?